agreements/project-contribution/asset-transfer-amendment-email.md

```diff
@@ -0,0 +1,51 @@
+# Email Template for Amendment to Asset Transfer Agreement
```

Comment:
I don't understand the use-case. 

I'm thinking of the Asset Transfer as a one-off activity... Company X -> CommonHaus.

Once the Assert Transfer is "executed" isn't it then just a historical record?  If Company X want to give more assets to CommonHaus wouldn't it just be a new Assert Transfer?

Reply:
This allows a more abbreviated amendment... i.e. you forgot a domain name or a mailing list.

Reply:
Ok.